tests/pthread/test_pthread_gcc_atomic_fetch_and_op.cpp

```diff
@@ -47,37 +47,37 @@ void *thread_fetch_and_sub(void *arg)
 	pthread_exit(0);
 }
 
-volatile int fetch_and_or_data = 0;
+volatile long fetch_and_or_data = 0;
```

Comment:
This seems like maybe it could be changing what we are testing here?

Reply:
I was assuming there's equivalent 64-bit atomic ops for this, any reason why not?